nasa/openmct
# [#5522] No Code Coverage for toggleClass utility function in src/utils/template/templateHelpers.js

Closes #5522 

### Describe your changes:
I have added tests that validate the `toggleClass` function in `src/utils/template/templateHelpers.js` works as expected by validating:
- What happens when various arguments are passed to this function (i.e., when is it expected to throw/not throw)
- That a class can be _added_ to an element, under various circumstances (i.e., when the element has no classes, a single class, or multiple classes already applied).
- That a class can be _removed_ from an element, under various circumstances (i.e., when the element has no classes, a single class, or multiple classes already applied).

#### To Test:
- Navigate to the new `src/utils/template/templateHelpersSpec.js` file
- Update line 32 to the following (prepending an `f` to `describe`)(this is to ensure no other test suites are run):
```javascript
fdescribe('toggleClass', () => {
```
- Run `npm test`
- See that all the new tests pass

### All Submissions:


### Author Checklist

* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Appropriate unit tests included?
* [ ] Commit messages meet standards?
